### PR TITLE
auth0: Add options for getConnections()

### PR DESCRIPTION
**Pull request by mamacdon**
This PR updates the `auth0` library type definitions. It adds missing type definitions for params to the `getConnections()` method.

-----------
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes

Example code:
https://github.com/auth0/node-auth0/blob/master/test/management/connections.tests.js#L127

- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

N/A, this is a fix to the type definitions

### CODE REVIEW

File: types/auth0/index.d.ts [hunk 1]
```diff
@@ -541,6 +541,23 @@ export interface CreateConnection extends UpdateConnection {
     strategy: Strategy;
 }
 
+export interface GetConnectionsOptions extends PagingOptions {
+    /** List of fields to include or exclude */
+    fields?: string[] | undefined;
```
Comment (peterblazejewicz) [inline types/auth0/index.d.ts:546]:
```suggestion
    fields?: string | string[] | undefined;
```
the test uses string literal (string value), not a list of strings

Comment (mamacdon) [reply types/auth0/index.d.ts]: I don't understand what test you are referring to…? But yes, you are right – both `string` and `string[]` are accepted here. I've updated the type.

Comment (peterblazejewicz) [reply types/auth0/index.d.ts]: > I don't understand what test you are referring to…? But yes, you are right – both `string` and `string[]` are accepted here. I've updated the type.

one from the posted link:
https://github.com/auth0/node-auth0/blob/master/test/management/connections.tests.js#L127

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@mamacdon thanks!

**Comment by peterblazejewicz** [comment]
Ready to merge

**Pull request merged**